packages/dx-react-scheduler-material-ui/src/templates/appointment-form/basic/layout.jsx

```diff
@@ -198,6 +199,22 @@ const LayoutBase = ({
         onValueChange={changeNotes}
         className={classes.notesEditor}
       />
+      {resources.map(resource => (
+        <>
```

Comment:
Seems that we need to add `key` here
![image](https://user-images.githubusercontent.com/51905811/68465656-ea864380-0223-11ea-9c5c-fabf6b798da6.png)